src/jortestkit/src/process/wait.rs

```diff
@@ -14,6 +15,24 @@ pub struct Wait {
     current: u64,
 }
 
+impl Display for Wait {
+    fn fmt(&self, f: &mut Formatter<'_>) -> std::fmt::Result {
+        if self.current != 0 {
+            write!(
+                f,
+                "Total attempts '{}' with sleep between for '{:?}'. Currently at {} attempt",
+                self.attempts, self.sleep, self.current
+            )
+        } else {
```

Comment:
do we need this `else`? Something like `"Total attempts 0 with sleep between for .... Currently at 0 attempts"` is pretty clear IMO? Doesn't hurt to have it, but this seems like a good spot for simplifying stuff a bit